internal/cli/utils.go

```diff
@@ -86,6 +86,62 @@ func parsePageRange(pageRange string) ([]int, error) {
 	return pages, nil
 }
 
+func parsePageRangeUnsorted(pageRange string) ([]int, error) {
+	var pages []int
+
+	rngs := strings.Split(removeSpaces(pageRange), ",")
+	for _, rng := range rngs {
+		if rng == "" {
+			continue
+		}
+
+		indices := strings.Split(rng, "-")
+
```

Comment:
could you remove this empty line?

Reply:
This function is the exact copy of the `parsePageRange` function along with its formatting. Should I remove from both functions or  only the line 99 one?

Reply:
sure, you could remove both of them.

Reply:
Done :+1: